### PR TITLE
styles: built Wishlist page: added navbar, and cards

### PR DESCRIPTION
**Pull request by Roshannair24**
added navbar, and cards with active heart ico
scrrenshot wishlist
n

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,27 @@
...
-    ecomm app
+
+
+<div   class="e-comm-container" style="background-color: aqua;">
+
+
+
+
+
+frgtg
```
Comment (spraveenofficial) [inline index.html]: Remove this unwanted space.

Comment (Roshannair24) [reply index.html]: okay, i will

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+  padding: 1rem;
+  font-size: 2rem;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+
+  color: var(--call-to-action);
+}
+
+.k-nav .k-nav-icon {
```
Comment (vaibhav18matere) [inline css/styles.css:47]: you can use better class-names, which are more descriptive

File: wishlist.html [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+      rel="stylesheet"
+      href="https://kundalini-ui.netlify.app/css/styles.css"
+    />
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="k-nav">
+      <h1>Iconic</h1>
+
```
Comment (vaibhav18matere) [inline wishlist.html:30]: you can give top margin to the title which will make UI more clean

**Pull request merged**